### PR TITLE
Parse the codecs string into separate entries so that we can create split audio/video buffers

### PR DESCRIPTION
**Pull request by imbcmdth**
These are the minimal set of changes required to support split audio and video source buffers.

### CODE REVIEW

**Comment by gkatsev** [comment]
The tests are failing because of spaces in the codec output. The expects have them and actuals don't.

File: src/videojs-media-sources.js [hunk 1]
```diff
@@ -216,10 +236,12 @@ addTextTrackData = function (sourceHandler, captionArray, metadataArray) {
       // append muxed segments to their respective native buffers as
       // soon as they are available
       this.transmuxer_ = new Worker(videojs.MediaSource.webWorkerURI || '/src/transmuxer_worker.js');
+      this.transmuxer_.postMessage({action:'init', options: {remux: false}});
```
Comment (imbcmdth) [inline src/videojs-media-sources.js]: This is *technically* the only thing that is required to get non-combined output!

Comment (dmlap) [inline src/videojs-media-sources.js]: Amazing!

**Comment by dmlap** [comment]
Beauty.

**Pull request merged**